### DISCUSSION TITLE
How could I set the fetch submodules --jobs= option?

### DISCUSSION THREAD

**Discussion by eki3z** [magit/magit#5144] [Support] [answered]
In the [manual](https://magit.vc/manual/magit/Submodule-Transient.html)  I find an option called `magit-fetch-modules-jobs` to customize --jobs option to fetch all modules, however there setting it has no effect, and when I call `magit-fetch-modules`, there isn't any transient window to customize it, did I miss something?

**Comment by kyleam** [comment, marked as answer]
That section of the manual should have been updated with ed806783 (Remove variables and functions obsolete since v3.3.0, 2023-03-13).  To set the `--jobs` option to something else, you can use a prefix argument (`C-u`) to before calling `magit-fetch-modules` bring up its transient where you can set (and save) the `--jobs` value.

**Comment by eki3z** [comment]
Oh ,thanks for reply ! I succeed to set jobs to 8. Is there any method to set it default to other value like 8 instead of C-u calling every time?

**Reply by kyleam** [reply]
You can save the transient value: https://magit.vc/manual/transient/Saving-Values.html

**Reply by eki3z** [reply]
That is exactly what I want, thanks again ! I think the manual should be updated to mark that option as obsoleted.
